fractal_server/app/schemas/v2/task_collection.py

```diff
@@ -89,6 +90,33 @@ def package_version_validator(cls, v, values):
         return v
 
 
+class TaskCollectCustomV2(BaseModel):
```

Comment:
Let's add a docstring here, which we can also duplicate as help messages in the webclient.

I'd go with
```
python_interpreter: Absolute path to the Python interpreter to be used for running tasks.
source: A common label identifying this package.
manifest: Manifest of a Fractal task package (this is typically the content of `__FRACTAL_MANIFEST__.json`).
version: Optional version of tasks to be collected.
package_name: Name of the package, as used for `import <package_name>`; this is then used to extract the package directory (`package_root`) via `pip show <package_name>`.
package_root: The folder where the package is installed. If not provided, it will be extracted via `pip show` (requires `package_name` to be set).
```